plugins/vpc-shared-eni/config/netconfig.go

```diff
@@ -40,23 +40,30 @@ type NetConfig struct {
 	GatewayIPAddress net.IP
 	InterfaceType    string
 	TapUserID        int
+	TaskENI          TaskENIConfig
 	Kubernetes       KubernetesConfig
 }
 
 // netConfigJSON defines the network configuration JSON file format for the vpc-shared-eni plugin.
 type netConfigJSON struct {
 	cniTypes.NetConf
-	ENIName          string   `json:"eniName"`
-	ENIMACAddress    string   `json:"eniMACAddress"`
-	ENIIPAddress     string   `json:"eniIPAddress"`
-	VPCCIDRs         []string `json:"vpcCIDRs"`
-	BridgeType       string   `json:"bridgeType"`
-	BridgeNetNSPath  string   `json:"bridgeNetNSPath"`
-	IPAddress        string   `json:"ipAddress"`
-	GatewayIPAddress string   `json:"gatewayIPAddress"`
-	InterfaceType    string   `json:"interfaceType"`
-	TapUserID        string   `json:"tapUserID"`
-	ServiceCIDR      string   `json:"serviceCIDR"`
+	ENIName          string        `json:"eniName"`
+	ENIMACAddress    string        `json:"eniMACAddress"`
+	ENIIPAddress     string        `json:"eniIPAddress"`
+	VPCCIDRs         []string      `json:"vpcCIDRs"`
+	BridgeType       string        `json:"bridgeType"`
+	BridgeNetNSPath  string        `json:"bridgeNetNSPath"`
+	IPAddress        string        `json:"ipAddress"`
+	GatewayIPAddress string        `json:"gatewayIPAddress"`
+	InterfaceType    string        `json:"interfaceType"`
+	TapUserID        string        `json:"tapUserID"`
+	ServiceCIDR      string        `json:"serviceCIDR"`
+	TaskENI          TaskENIConfig `json:"taskENI"`
+}
+
+// TaskENI defines the Task Networking specific data required by the plugin
+type TaskENIConfig struct {
+	Enable bool `json:"enableTaskENI"`
```

Comment:
`Enable bool` is not conveying intent clearly. 
Can this be renamed to `PauseContainer` to indicate the infra container in use?